tinyagentos/routes/userspace_apps.py

```
@@ -0,0 +1,203 @@
+_MAX_PACKAGE_BYTES = 64 * 1024 * 1024
+
+
+@router.post("/api/userspace-apps/install")
+async def install_app(request: Request, package: UploadFile | None = File(default=None)):
+    store = request.app.state.userspace_apps
+    if package is not None:
+        data = await package.read(_MAX_PACKAGE_BYTES + 1)
+    else:
+        try:
+            body = await request.json()
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)